terraform/domain-redirects/redirects.tf

```diff
@@ -102,6 +102,7 @@ module "docs_rs_metadata" {
   to_path = "about/metadata"
   from = [
     "package.metadata.docs.rs",
+    "project.metadata.docs.rs",
```

Comment:
Unsure what the intention of this is but does this elevate the usage of a feature is is legacy / deprecated?   We don't even document this in the [manifest documentation](https://doc.rust-lang.org/cargo/reference/manifest.html) and many of us have gone a long time without even knowing it exists (preparing the PR to merge `cargo add` into `cargo` was when I learned of it after all of my other `cargo add` work).